codalab/lib/bundle_cli.py

```diff
@@ -1924,14 +1926,21 @@ def do_search_command(self, args):
             params={'worksheet': worksheet_uuid, 'keywords': args.keywords, 'include': ['owner']},
         )
 
+        print('Note: at most 10 results will be shown.')
```

Comment:
Make the `10` filled in from a variable that governs the actual logic as opposed to hard-coded.

Reply:
Good point. I moved it to a const.